### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please don't use git clone or `ldd`. Is `AFL_NO_FORKSRV` necessary here?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+image: ghcr.io/capuanob/xhttp-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image name. Tests has been changed to testsuite.

**Review by xansec — CHANGES_REQUESTED** [review]
It seems like you've uploaded quite an extensive corpus. Is this necessary? Usually 1-2 seeds will suffice. Please make the requested changes.

**Comment by xansec** [comment]
I would also not upload such a large corpus.

File: Dockerfile (hunk 1 above)
Comment (capuanob) [reply Dockerfile:1]: I had to use AFL_NO_FORKSRV as every run would crash due to an inability to spool up the fork server.

Comment (capuanob) [reply Dockerfile:1]: Nevermind, did something change with the fork server problem we originally discussed? Re-enabled that and it is blazing fast now

**Comment by capuanob** [comment]
@xansec Done! With the reduced corpus, this only hit a test case size of 6 but a smaller edge coverage. If given more time, it would have been higher upon discovering those edge cases

**Review by xansec — APPROVED** [review]
Yes, we have made some fixes/improvements on the backend. LGTM!
